### PR TITLE
Correcting minikube quickstart steps

### PR DESCRIPTION
**Pull request by rajeshdeshpande02**
In minikube quickstart section , it was difficult to differentiate between command and its output. Also, There was no details given for each steps and its expected output. This PR fixes these issues.

### CODE REVIEW

File: content/en/docs/setup/minikube.md [hunk 1]
```diff
@@ -45,103 +45,130 @@ Note that the IP below is dynamic and can change. It can be retrieved with `mini
...
+        ```shell        
+        kubectl run hello-minikube --image=k8s.gcr.io/echoserver:1.10 --port=8080
+        ```        
+   The output of a successful command verifies that the deployment is created:
+   
+        ```
+        deployment.apps/hello-minikube created
+        ```
+        
+1. Expose a echoserver deployment to create service
```
Comment (FranklinYu) [inline content/en/docs/setup/minikube.md]: *an* echo server?

File: content/en/docs/setup/minikube.md [hunk 2]
```diff
@@ -45,103 +45,130 @@ Note that the IP below is dynamic and can change. It can be retrieved with `mini
...
-```
-Stopping local Kubernetes cluster...
-Stopping "minikube"...
-```
+1. Start a minikube 
+
+        ```shell        
+        minikube start
+        ```    
+   The output displays the kubernetes cluster is started:
```
Comment (FranklinYu) [inline content/en/docs/setup/minikube.md]:
```suggestion
   The output displays that the kubernetes cluster is started:
```

File: content/en/docs/setup/minikube.md [hunk 3]
```diff
@@ -45,103 +45,130 @@ Note that the IP below is dynamic and can change. It can be retrieved with `mini
...
+   
+        ```
+        Starting local Kubernetes cluster...
+        Running pre-create checks...
+        Creating machine...
+        Starting local Kubernetes cluster...
+        ```
+        
+1. Create a echoserver deployment
```
Comment (FranklinYu) [inline content/en/docs/setup/minikube.md]:
```suggestion
1. Create an echo server deployment
```

**Comment by rajeshdeshpande02** [comment]
Thank you @FranklinYu for your quick review.

**Comment by tengqm** [comment]
The output preview is here: https://deploy-preview-14355--kubernetes-io-master-staging.netlify.com/docs/setup/minikube/

It looks like code snippet inside a list has some strange behaviors.

**Comment by rajeshdeshpande02** [comment]
@tengqm  Thanks for your review. Can you please give more details about issues you are seeing for this change. I am not able to figure it about.

**Comment by FranklinYu** [comment]
@rajeshdeshpande02 You either use back-tick fence, or indent 4 spaces further. Not both.

**Comment by rajeshdeshpande02** [comment]
@FranklinYu @tengqm Corrected issues you were mentioned in review. Can you please have a look at it ?

**Pull request merged**